src/components/Calculator.js

```diff
@@ -1,21 +1,79 @@
-import React from 'react';
+import React, { useState } from 'react';
 import PropTypes from 'prop-types';
 import '../styles/calculator.css';
+import Screen from './Display';
+import calculate from '../logic/calculate';
 
-function Button({ children }) {
+export function Button({ children, onClick }) {
```

Comment:
### Optional
- [ ] It would be nice if we move the Button component into a separate file, this will make the code more readable